Updated the change log to include SSLv3 being disabled by default.

bog-the-frog opened the pull request:
Updated changelog to include SSLv3 being disabled by default from version 1.1.0 onward

This change was implemented into version 1.1.0 and onward. The last version that had SSLv3 enabled was version 1.0.2h. The person who made this change was Kurt Roeckx, so he has been noted as the contributor in the changelog.
This is resolving issue [8969](https://github.com/openssl/openssl/issues/8969)

CLA: trivial

<!--
Thank you for your pull request. Please review these requirements:

Contributors guide: https://github.com/openssl/openssl/blob/master/CONTRIBUTING.md

Other than that, provide a description above this comment if there isn't one already

If this fixes a GitHub issue, make sure to have a line saying 'Fixes #XXXX' (without quotes) in the commit message.
-->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
- [x] documentation is added or updated
- [ ] tests are added or updated

bog-the-frog committed:
Updated the change log to include SSLv3 being disabled by default.

This change was implemented into version 1.1.0 and onward. The last version that had SSLv3 enabled was version 1.0.2h, which is why the addition was made where it is.

bog-the-frog commented:
@bbbrumley
@Jormungandr358

bbbrumley commented:
Hi, this looks good but you'll want to reference the OpenSSL issue in the actual git commit message, not just here in the web UI. Also the same with "CLA: trivial" -- it needs to be in the commit message, on a line by itself

[kind of like this](https://github.com/openssl/openssl/commit/c2598e3157ce583ef3b3c5aed431cfafcf44ed46.patch)

Cheers

bog-the-frog committed:
Updated the change log to include SSLv3 being disabled by default.

Fixes #8969

This change was implemented into version 1.1.0 and onward. The last version that had SSLv3 enabled was version 1.0.2h, which is why the addition was made where it is.

CLA: trivial

bog-the-frog committed:
Merge branch 'bog-the-frog' of https://github.com/bog-the-frog/openssl into bog-the-frog

bog-the-frog committed:
Merge pull request #3 from openssl/master

update

Jormungandr358 commented:
With the addition of the CLA this is well done. A clear message that addresses an important change regarding the default disabling of SSLv3. Also, nice work adding the original contributor in the changelog.
